### PR TITLE
Feat: Added Text Utilities and Landing Page

### PR DESCRIPTION
**Pull request by JaswanthReddyPadala**
Added : 
- Landing Page and Docs Page Layout
- Text Utilities Component

Preview Deploy : [https://quik-ui-text-utilities-preview.netlify.app/components/typography/typography.html](url)

### CODE REVIEW

File: components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+.text-center{ text-align: center; }
+
+.text-right{ text-align: right; }
+
+/* text-decoration */
+.text-underline { text-decoration: underline; }
+
+.text-normal {
+    text-decoration: none;
+    color: #181818;
```
Comment (anjali1102) [inline components/typography/typography.css]: can we use color variable here ?

File: components/typography/typography.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                        <p class="text-right">Right Text</p>
+                    </script>
+                </div>
+            </div>
+
+
+            <div class="docs-content-sub-container">
+                <h1 class="text-24 mt-1">Text Decoration</h1>
+                <div class="flex flex-col align-items-baseline width-max m-1">
+                    <div class="flex text-16 text-decoration-container">
```
Comment (anjali1102) [inline components/typography/typography.html:134]: suggestion: short classnames
for future

File: quik.css [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+::-webkit-scrollbar-track {
+  background: #f1f1f1;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #888;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: #555;
```
Comment (anjali1102) [inline quik.css]: color variable

**Review by anjali1102 — CHANGES_REQUESTED** [review]
code formatting

File: components/typography/typography.html (hunk 2 above)
Comment (JaswanthReddyPadala) [reply components/typography/typography.html:134]: Okay Anjali

File: quik.css (hunk 3 above)
Comment (JaswanthReddyPadala) [reply quik.css]: Done, Please check

**Review by samyakshah3008 — APPROVED** [review]
Good naming conventions 
Clean UI and code
Good for approval ✅

**Review by anjali1102 — APPROVED** [review]

**Pull request merged**